Gemfile

```diff
@@ -43,6 +43,9 @@ gem 'bcrypt', '~> 3.1.11'
 # Ruby wrapper for Wunderground APIs
 gem 'wunderground'
 
+# Figaro to secure configuration values
+gem "figaro"
```

Comment:
I didn't know about figaro and didn't realize what it does when I suggested `dotenv` earlier.